src/components/player/cue-bar.tsx

```diff
@@ -49,24 +53,65 @@ export default class CueBar extends React.Component<any> {
   }
 
   render() {
+    console.log('this.state: ', this.state)
     const {className, disableCompletely, children, player} = this.props as any
     const {duration} = player
 
     return disableCompletely ? null : (
       <div className={classNames('cueplayer-react-cue-bar', className)}>
         {this.state?.cues?.notes?.map((noteCue: any) => {
           return (
-            <NoteCue key={noteCue.text} cue={noteCue} duration={duration} />
+            <NoteCue
+              key={noteCue.text}
+              cue={noteCue}
+              duration={duration}
+              currentCuePopup={this.state.currentCuePopup}
+              setCurrentCuePopup={this.setCurrentCuePopup.bind(this)}
+            />
           )
         })}
       </div>
     )
   }
 }
 
-const NoteCue: React.FC<any> = ({cue, duration, className}) => {
+const CuePopup: React.FC<any> = ({note, popupPersists, currentCuePopup}) => {
+  return note?.title === currentCuePopup || popupPersists ? (
+    <div
+      className="absolute w-40 min-h-[4rem] rounded-md bg-white p-3 text-xs top-0 left-0 z-10 text-black border border-gray-400"
```

Comment:
when in the context of the player I think being consistent with the player CSS styles would be beneficial

same structure, same naming conventions, use of global css, etc

this means no Tailwind or emotion

this isn't critical, but I think it will also make the player work more portable for us so easier to just do it now instead of later